processor/src/chiplets/ace/mod.rs

```diff
@@ -1,5 +1,3 @@
-#![allow(dead_code)]
-
```

Comment:
It isn't necessary to apply this flag to the whole module. The only unused thing was the `raw_circuit_hash` method